website/registrations/templates/registrations/email/renewal_accepted.txt

```diff
@@ -7,14 +7,15 @@ you get access to everything Thalia has to offer once again: paying your members
 
 The membership fees are €{{ fees }}. Those can be paid in a few ways:
 
-1. You can let us withdraw the money from your bank account via Thalia Pay. For this, you
-will need to have a bank account added to your account and have signed a direct debit mandate.
-To pay with Thalia Pay, go to {{ url }}.
+1. You can pay using debit card or cash by visiting the Thalia board room (M1.0.08) during the lunch break (12:15-13:15).
 
 2. You can transfer the amount of €{{ fees }} to NL 23 INGB 0006 2341 84 with the description "Contribution {{ name }}".
 Once your payment is processed by us, you will receive a confirmation email.
-
-3. You can pay using debit card or cash by visiting the Thalia board room (M1.0.08) during the lunch break (12:15-13:15).
+{% if thalia_pay_enabled %}
+3. You can let us withdraw the money from your bank account via Thalia Pay. For this, you
+will need to have a bank account added to your account and have signed a direct debit mandate.
+To pay with Thalia Pay, go to {{ url }}.
+{% endif %}
```

Comment:
I think that Thalia Pay should be the first / preferred option if it's available.